server/prospect_campaign_join_resources.py

```diff
@@ -0,0 +1,249 @@
+from typing import List, Any
```

Comment:
I think you copy and pasted all of Users in here, but isn't this supposed to be prospect campaign?

Reply:
Those were supposed to be in a different branch. Oops! Fixed now.